src/setup.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const { User } = require('./models/users');
+const { Expense } = require('./models/expenses');
+
+User.sync({ force: true });
+Expense.sync({ force: true });
```

Comment:
Using 'force: true' in the sync method will cause the table to be dropped and recreated every time the application is started. This will result in a loss of data. Consider removing the 'force' option or setting it to false.